Bottom: 296f1c2066c8f8c7729ae3381286d6e689b1d40c
Top:    acb90057a5c2227df4d480223275c85609da35d2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 23:53:32 +0100

hid/{ps,eps,gerber}: Clear CHECKPLANESFLAG before exporting

Ensure we are in a sensible view state before exporting. This
update those HIDs which cleared other similar renedering flags
before exporting to include the CHECKPLANES flag.


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 4a2d233..1539f88 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -521,6 +521,7 @@ gerber_do_export (HID_Attr_Val * options)
   save_thindraw = PCB->Flags;
   CLEAR_FLAG(THINDRAWFLAG, PCB);
   CLEAR_FLAG(THINDRAWPOLYFLAG, PCB);
+  CLEAR_FLAG(CHECKPLANESFLAG, PCB);
 
   if (!options)
     {
diff --git a/src/hid/ps/eps.c b/src/hid/ps/eps.c
index 560f897..a406f38 100644
--- a/src/hid/ps/eps.c
+++ b/src/hid/ps/eps.c
@@ -198,6 +198,7 @@ eps_hid_export_to_file (FILE * the_file, HID_Attr_Val * options)
   save_thindraw = PCB->Flags;
   CLEAR_FLAG(THINDRAWFLAG, PCB);
   CLEAR_FLAG(THINDRAWPOLYFLAG, PCB);
+  CLEAR_FLAG(CHECKPLANESFLAG, PCB);
 
   f = the_file;
 
diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 87202f6..8e29fd5 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -601,6 +601,7 @@ ps_hid_export_to_file (FILE * the_file, HID_Attr_Val * options)
   save_thindraw = PCB->Flags;
   CLEAR_FLAG(THINDRAWFLAG, PCB);
   CLEAR_FLAG(THINDRAWPOLYFLAG, PCB);
+  CLEAR_FLAG(CHECKPLANESFLAG, PCB);
 
   global.f = the_file;
   global.drill_helper = options[HA_drillhelper].int_value;
